python/packages/ai/teams/ai/promptsv2/text_section.py

```diff
@@ -0,0 +1,102 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+
+from typing import List
+
+from botbuilder.core import TurnContext
+
+from ...state import Memory
+from ..tokenizers import Tokenizer
+from .message import Message
+from .prompt_functions import PromptFunctions
+from .prompt_section_base import PromptSectionBase
+from .rendered_prompt_section import RenderedPromptSection
+
+
+class TextSection(PromptSectionBase):
+    """
+    A section of text that will be rendered as a message.
+
+    Attributes:
+        text (str): Text to use for this section.
+        role (str): Message role to use for this section.
+
+    """
+
+    _text: str
+    _role: str
+    _length: int
+
+    # pylint: disable=too-many-arguments # No argument can be removed based on the design
```

Comment:
Not Blocking: I am going to change this lint rule so we don't need to keep enabling/disabling it like this